### ISSUE TITLE
GtalkSMS creates the same MCU room every time

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#58]
```
Just found myself in a MUC room who is sending SMS to 2 contacts of mine.

In createRoom() in XmppManager
String cnx = "GTalkSMS_" + _rand + "_" + _settings.login.replaceAll("@", "_") 
            + "@" + _settings.mucServer; 

all var's are final (per xmpp) session, as _rand is only initalized at creation 
of the XmppManager class. So all created rooms have the same JID/room ID.

Just wondering if this has ever worked before? Never noticed this. Anyway here 
is the patch. (We could add a collision protection at some time in the future).
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 21 Jan 2011 at 11:30

Attachments:
- [muc_room_rand.patch](https://storage.googleapis.com/google-code-attachments/gtalksms/issue-58/comment-0/muc_room_rand.patch)

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 21 Jan 2011 at 7:17
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
No, has not worked for me with two different people. It even sent the whole log 
that I just had with one person, to the other person that just texted me. It 
was kind of awkward because I just had a whole conversation with one person and 
that was just sent to the other who just texted me (and joined the room). 
```

Original comment by `i.vita...@gmail.com` on 21 Jan 2011 at 10:18

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 4:13
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.7.3
```

Original comment by `Florent....@gmail.com` on 22 Jan 2011 at 8:07
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:51
- Changed state: **Deployed**
